libs/shared/theme/src/theme.tsx

```diff
@@ -21,6 +21,8 @@ export const theme = extendTheme({
         divider: '#101113',
         background: {
           paper: '#1E1F25',
+          paperHeader: '#23242A',
+          paperFooter: '#23242A',
```

Comment:
I still think that having twice the same value with different keys makes very little sense. Maybe a more generic `section` field...